docs/_extra/api-reference/hypothesis-v1.yaml

```diff
@@ -111,6 +111,7 @@ tags:
   - name: general
   - name: annotations
   - name: groups
+  - name: memberships
```

Comment:
The membership APIs used to be listed under groups in the docs. I'm moving them out into their own separate section since they do form a separate set of endpoints (list-memberships, add-membership, get-membership, edit-membership and delete-membership). Like annotation, user and group, "membership" is now a first-class noun in the h model and API.